app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -175,15 +153,27 @@ def self.reserve_port(ports)
       end
     end
 
-    def command_line_params
-      params = {}
-      params[:port] = port if port.kind_of?(Numeric)
+    def rails_server_options
+      self.class.configure_secret_token
```

Comment:
Before we load the ui/ws worker in the forked child, configure/reset the secrets and set the token.